qtrees/data_processor.py

```diff
@@ -0,0 +1,311 @@
+import pandas as pd
+import numpy as np
+from functools import reduce
+import datetime as dt
+from typing import Union, Optional
+from sklearn.preprocessing import OrdinalEncoder
+
+ROLLING_WINDOW = 7
+FORECAST_HORIZON = 14
+AUTOREGRESSIVE_LAG = 3
+
+class Data_loader:
```

Comment:
Besser: DataLoader ("camel case"). Ist in Python üblicher. Daher gern "refactorn"